tensorflow/workspace.bzl

```diff
@@ -508,6 +508,14 @@ def tf_workspace(path_prefix="", tf_repo_name=""):
   )
 
   native.new_http_archive(
+    name = "lmdb",
```

Comment:
@bowang It looks like you also need it to add in a couple of build rules here:

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/lib_package/BUILD